metrics.md

```diff
@@ -0,0 +1,85 @@
+# Universal Search Metrics
+
+A summary of the metrics the Universal Search add-on and recommendation server will record.
+
+**Definitions:**
+
+- **URL bar** — the unified address + search bar at the top of the browser.
+- **Popup** — the popup that appears when you begin typing into the URL bar.
+- **Search** — the full lifecycle, beginning when the user types a key into the url bar, and ending when the user either chooses an item from or otherwise dismisses the popup.
+
+
+## Client metrics
+
+Metrics gathered from the Universal Search add-on will be reported to the Test Pilot add-on, which will augment that information and relay it to Firefox's telemetry system via `submitExternalPing`. This is not currently implemented on the Test Pilot side; those efforts are encompassed by [issue #234](https://github.com/mozilla/testpilot/issues/234).
+
+
+### Events
+
+Universal Search will record and report each of the following events:
+
+- User selects a recommendation.
```

Comment:
Yeah I agree that it could be a single event like selected/navigated/closed (whatever the right way to think about this) with these attributes attached to it.